history.go

```diff
@@ -51,6 +52,14 @@ func (h *History) Newer(buf *Buffer) (new *Buffer, changed bool) {
 	return new, true
 }
 
+// Get x lines back in the history as a string array
+func (h *History) Get(lines int) []string {
```

Comment:
[imo] I have a little concerned about this method name/

* We don't imagine that this method returns the histories from the last? or from we selected?
* When we want to add another method to retrieve *one* history, this name will conflict.